### PR TITLE
#3 About soiree and request modal conflicts resolved

### PR DESCRIPTION
**Pull request by shubhamnautiyal**

### CODE REVIEW

File: public/wp-content/themes/ColoredCow/functions.php [hunk 1]
```diff
@@ -2,11 +2,10 @@
 
 if ( ! function_exists( 'cc_scripts' ) ) {
     function cc_scripts() {
-        wp_enqueue_script('cc-bootstrap', get_template_directory_uri().'/dist/lib/js/bootstrap.min.js', array('jquery'), '1.0.0', true);
-        wp_enqueue_script('main', get_template_directory_uri().'/main.js', array('jquery', 'cc-bootstrap'), '1.0.0', true);
+        
+        wp_enqueue_script('cc-jquery-script','https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js');
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/functions.php]: please use WP's inbuilt jquery.

File: public/wp-content/themes/ColoredCow/functions.php [hunk 2]
```diff
@@ -2,11 +2,10 @@
...
         wp_enqueue_script('cc-bootstrap-script','https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js');
         wp_enqueue_script('cc-custom-fontawesome','https://use.fontawesome.com/ffc2c94a85.js');
-        wp_enqueue_script('cc-jquery-script','https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js');
         wp_localize_script( 'main', 'PARAMS', array('ajaxurl' => admin_url('admin-ajax.php')) );
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/functions.php]: what does it do? is there a specific purpose why you've put it here? Please read the documentation for wp_localize_script.

File: public/wp-content/themes/ColoredCow/functions.php [hunk 3]
```diff
@@ -17,7 +16,9 @@ function cc_scripts() {
     function cc_styles() {  
         wp_enqueue_style('cc-bootstrap', get_template_directory_uri().'/dist/lib/css/bootstrap.min.css');
         wp_enqueue_style('cc-bootstrap-css','https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css');
-        wp_enqueue_style('cc-custom-font','https://fonts.googleapis.com/css?family=Oswald');
+        wp_enqueue_style('cc-custom-font1','https://fonts.googleapis.com/css?family=Oswald');
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/functions.php]: fix the names please. what does cc-custom-font1 represent?

Comment (rathorevaibhav) [reply public/wp-content/themes/ColoredCow/functions.php]: Also, you can call multiple fonts within a single request. Here you are making 3 calls to google API and your page will take more time to load. Read on how you can load multiple fonts from google cdn in a single request.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 4]
```diff
@@ -5,9 +5,88 @@
     get_header();
 ?>
 
-<?php
-?>
- 
+<body>
+	<div class="container">
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php:9]: your indentation should be a single tab.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 5]
```diff
@@ -5,9 +5,88 @@
...
+		<div class="row">
+			<div class="col col-sm-12 col-md- col-lg-6">
+				<h2 class="about">Soiree</h2>
+				<br>
+			  	<p class="lead">ColoredCow celebrates every first Saturday of the month with family and friends. This custom has been started to take a little time off from work and enjoy some moments in life. we believe in sharing moments and learning with each other. Come and join us over music, food, drinks and some moments full of laughter and joy.</p>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: what does lead represent? Use a better class naming.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 6]
```diff
@@ -5,9 +5,88 @@
...
 ?>
 
-<?php
-?>
- 
+<body>
+	<div class="container">
+		<div class="row">
+			<div class="col col-sm-12 col-md- col-lg-6">
+				<h2 class="about">Soiree</h2>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: this is a title. I don't think it has any connection with **about**, right? Use a better name.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 7]
```diff
@@ -5,9 +5,88 @@
     get_header();
 ?>
 
-<?php
-?>
- 
+<body>
+	<div class="container">
+		<div class="row">
+			<div class="col col-sm-12 col-md- col-lg-6">
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: **col-md-** ??????? Also, what does col signify? is it a valid class?

File: public/wp-content/themes/ColoredCow/landing.php [hunk 8]
```diff
@@ -5,9 +5,88 @@
...
+				<h2 class="about">Soiree</h2>
+				<br>
+			  	<p class="lead">ColoredCow celebrates every first Saturday of the month with family and friends. This custom has been started to take a little time off from work and enjoy some moments in life. we believe in sharing moments and learning with each other. Come and join us over music, food, drinks and some moments full of laughter and joy.</p>
+				<hr>
+				<center>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: use CSS to make it center aligned. Remove this tag.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 9]
```diff
@@ -5,9 +5,88 @@
...
+			  		<div class="col-md-8 col-xs-12">
+						<h5 class="request">Wanna join the party?</h5>
+						<br>
+						<button type="button" class="btn btn-outline-warning btn-lg btn-block" data-toggle="modal" data-target="#exampleModal" data-whatever="@mdo">Request Invite</button>		
+					</div>
+				</center>		
+			</div>	
+		</div>
+		<div class="container">
+            <div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: exampleModal?? really? 😆  Rename.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 10]
```diff
@@ -5,9 +5,88 @@
...
+                  <div class="modal-dialog" role="document">
+                    <div class="modal-content">
+                          <div class="modal-header">
+                            <h3 class="modal-title" id="exampleModalLabel">New Request</h3>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: fix the id here.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 11]
```diff
@@ -5,9 +5,88 @@
...
+                            <button type="button" class="close" data-dismiss="modal" aria-label="Close" id="close"><span aria-hidden="true">&times;</span></button>
+                          </div>
+                         <form id="request_form">
+                              <div class="modal-body">
+                                  <p id="msg2"></p>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: what's msg2?

File: public/wp-content/themes/ColoredCow/landing.php [hunk 12]
```diff
@@ -5,9 +5,88 @@
...
+                                <input type="hidden" class="form-control" name="request_id" id="request_id">
+                                <div class="form-group">
+                                    <label for="recipient-name" class="form-control-label">
+                                        <h6>Your Name:</h6>
+                                    </label>
+                                    
+                                        <input type="text" class="form-control" placeholder="" name="request_name" id="request_name" maxlength="30" required>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: Remove the placeholder attribute if not using. This is only making your HTML statement longer and difficult to read.

Comment (rathorevaibhav) [reply public/wp-content/themes/ColoredCow/landing.php]: Attribute required should never be used as such. It should be required="required". every attribute must have a value assigned.

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 13]
```diff
@@ -34,16 +34,57 @@ Text Domain: coloredcow
 	}
 }
 
+.btn:hover
+{
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: this bracket should be in line 37. Please use the same format everywhere.

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 14]
```diff
@@ -34,16 +34,57 @@ Text Domain: coloredcow
...
+
 	background-image: url('dist/img/bg.png');
 	background-repeat: no-repeat;
 }
 
+.about{
+	padding-left: 22px;
+	font-family: Marcellus SC;
+	font-size: 85px; 
+	color:goldenrod;
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: color should be used as variables. Define them in variables.scss.

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 15]
```diff
@@ -34,16 +34,57 @@ Text Domain: coloredcow
...
+	text-align: justify;
+}
+
+.lead{
+	padding-left: 22px;
+	font-style: inherit;
+	font-size: 20px;
+	color: white;
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: color should be a variable.

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 16]
```diff
@@ -34,16 +34,57 @@ Text Domain: coloredcow
...
+	text-align: justify;
+}
+
+.request{
+	padding-top: 10px;
+	font-family: 'Marcellus SC'; 
+	font-size: 23px; 
+	color:white;
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: variable.

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 17]
```diff
@@ -34,16 +34,57 @@ Text Domain: coloredcow
...
+	align-content: center;
+}
+
+.modal-header{
+	font-family: 'Marcellus SC'; 
+	font-size: 25px;
+}
+
+.modal-body{
+	background-color: whitesmoke;
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: variable.

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 18]
```diff
@@ -34,16 +34,57 @@ Text Domain: coloredcow
...
+}
+
 .adminbtn{
 	font-family: 'Oswald'; 
 	font-size: 17px;
 }
 
 body{
+	padding-top: 100px;
+
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: why an empty line in between?

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 19]
```diff
@@ -34,16 +34,57 @@ Text Domain: coloredcow
 	}
 }
 
+.btn:hover
+{
+	cursor: pointer;
+	color: grey;
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: use as variable.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 20]
```diff
@@ -5,9 +5,88 @@
...
+                              </div>
+                              <div class="modal-footer pull-left">
+                              <button type="button" class="btn btn-success btn-lg"  id="Submit_request">Request</button>
+                              </div>
+                        </form> 
+                    </div>
+                </div>
+            </div>
+        </div>
+        <br>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: let's not use <br> anymore for styling. use div's margin/padding for spacing and alignment.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 21]
```diff
@@ -5,9 +5,88 @@
...
+                                    <label for="recipient-name" class="form-control-label">
+                                        <h6>Your Name:</h6>
+                                    </label>
+                                    
+                                        <input type="text" class="form-control" placeholder="" name="request_name" id="request_name" maxlength="30" required>
+                                    
+                                 </div>
+                                <div class="form-group">
+                                    <label for="recipient-name" class="form-control-label">
+                                        <h6>Your Email:</h6>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: why h6 inside the label? is it just for font size? you can assign font-size directly to the label.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 22]
```diff
@@ -5,9 +5,88 @@
...
+	<div class="container">
+		<div class="row">
+			<div class="col col-sm-12 col-md- col-lg-6">
+				<h2 class="about">Soiree</h2>
+				<br>
+			  	<p class="lead">ColoredCow celebrates every first Saturday of the month with family and friends. This custom has been started to take a little time off from work and enjoy some moments in life. we believe in sharing moments and learning with each other. Come and join us over music, food, drinks and some moments full of laughter and joy.</p>
+				<hr>
+				<center>
+			  		<div class="col-md-8 col-xs-12">
+						<h5 class="request">Wanna join the party?</h5>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: class="request" looks like an action. If it is just for the text, use a better name like request-question or request-text.

Let's make it a bit more professional. wanna -> Want to

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]
Phew! So many feedback. Have a look at all of them and let me know if you have any doubts.
